lib/Backend/Security.cpp

```diff
@@ -426,7 +426,7 @@ Security::EncodeAddress(IR::Instr *instr, IR::Opnd *opnd, size_t value, IR::RegO
     instrNew = LowererMD::CreateAssign(regOpnd, opnd, instr);
 
     size_t cookie = (size_t)Math::Rand();
```

Comment:
>cookie [](start = 11, length = 6)

nit: Please check if TyMachReg is ok when the value is an unsigned int. I am not sure about the implications.